src/service/resources-impl.js

```diff
@@ -487,6 +487,7 @@ export class Resources {
       resource.requestMeasure();
       dev().fine(TAG_, 'resource reused:', resource.debugid);
     } else {
+      this.relayoutAll_ = true;
```

Comment:
Did you see my optimize above? We could definitely go this route, though it'll add some latency between connect and remeasure.